spec/requests/api/v1/provider_spec.rb

```diff
@@ -174,9 +227,7 @@ def get_provider_codes_from_body(body)
                   },
                   {
                     'type' => 'application_alert_recipient',
-                    'name' => 'Application Alert Recipient Contact A123',
-                    'email' => 'application_alert_recipient@acmescitt.education.uk',
-                    'telephone' => '020 812 345 678'
+                    'email' => 'application_alert_recipient@acmescitt.education.uk'
```

Comment:
I feel like it'd be more cautious to list `email` and `telephone` here, but just have them set to empty strings.

Reply:
Done 👍